### DISCUSSION TITLE
Object level Security

### DISCUSSION THREAD

**Discussion by ShilpaMohanan2020** [TabularEditor/TabularEditor3#1180] [Data Modeling]
### Description

I wanted to hide 2 columns from the Power BI reports for a specific user. For that I created a role in the modelling tab and makes the DAX empty. Then the tabular editor 3 , in the tables , I select the perticular 2 colums and made OLS none from default. In the power BI security I add the secific user to the role. But the user is not able to view the entire table.

### Tabular Editor 3 Version

Tabular Editor 3, 64bit

### Screenshots

![sc1](https://github.com/TabularEditor/TabularEditor3/assets/123121814/debc122c-afb8-485a-8660-61258b9bf210)
![sc2](https://github.com/TabularEditor/TabularEditor3/assets/123121814/cae61dba-e53e-449e-a22f-1d9c1dda96a9)

### Expected behavior

Fields that need to be fixed

Something's wrong with one or more fields: (Users) User First Name, (Users) User Last Name
Please try again later or contact support. If you contact support, please provide these details.
See details
[Contact support](https://support.powerbi.com/)
Close

### Crash Report

Fields that need to be fixed

Something's wrong with one or more fields: (Users) User First Name, (Users) User Last Name
Please try again later or contact support. If you contact support, please provide these details.
See details
[Contact support](https://support.powerbi.com/)
Close

### Windows Version

3

**Comment by mlonsk** [comment]
I am moving this to our discussion forum as it is not an issue or bug with regard to Tabular Editor 3.

**Comment by mlonsk** [comment]
Hi @ShilpaMohanan2020 
Object Level Security blocks the user from seeing columns in visuals such as tables. The visual will not render for the user as the two columns (User First Name & User Last Name) are not availably. You can see that the same happens in this video where [SQLBI explores Object Level Security](https://youtu.be/8s3MByrZJgg?t=324)
You will have to remove the columns from the table visual for the user to see them, perhaps making two different version of the table depending on who is viewing it.
